### PR TITLE
[ko] Update outdated files in dev-1.21-ko.2 (p1)

### PR DESCRIPTION
**Pull request by jihoon-seo**
Related issue: #27764

> ### 20 files to be modified
> 1. [ ]  M1.  `content/en/case-studies/ibm/ibm_featured_logo.svg`  | 1(+XS) 1(-)
> 2. [ ]  M2.  `content/en/docs/concepts/architecture/nodes.md`  | 7(+XS) 0(-)
> 3. [ ]  M3.  `content/en/docs/concepts/cluster-administration/system-metrics.md`  | 1(+XS) 0(-)
> 4. [ ]  M4.  `content/en/docs/concepts/configuration/configmap.md`  | 1(+XS) 1(-)
> 5. [ ]  M5.  `content/en/docs/concepts/containers/container-lifecycle-hooks.md`  | 1(+XS) 2(-)
> 6. [ ]  M6.  `content/en/docs/concepts/containers/runtime-class.md`  | 3(+XS) 2(-)
> 7. [ ]  M7.  `content/en/docs/concepts/extend-kubernetes/_index.md`  | 8(+XS) 9(-)
> 8. [ ]  M8.  `content/en/docs/concepts/overview/working-with-objects/labels.md`  | 2(+XS) 3(-)
> 9. [ ]  M9.  `content/en/docs/concepts/scheduling-eviction/assign-pod-node.md`  | 1(+XS) 1(-)
> 10. [ ]  M10.  `content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md`  | 2(+XS) 2(-)
> 11. [ ]  M11.  `content/en/docs/concepts/services-networking/ingress.md`  | 1(+XS) 1(-)
> 12. [ ]  M12.  `content/en/docs/concepts/services-networking/service.md`  | 10(+S) 4(-)
> 13. [ ]  M13.  `content/en/docs/concepts/storage/volumes.md`  | 3(+XS) 4(-)
> 14. [ ]  M14.  `content/en/docs/concepts/workloads/controllers/job.md`  | 2(+XS) 2(-)
> 15. [ ]  M15.  `content/en/docs/concepts/workloads/controllers/replicaset.md`  | 1(+XS) 1(-)
> 16. [ ]  M16.  `content/en/docs/concepts/workloads/pods/disruptions.md`  | 1(+XS) 1(-)
> 17. [ ]  M17.  `content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md`  | 1(+XS) 1(-)
> 18. [ ]  M18.  `content/en/docs/contribute/new-content/open-a-pr.md`  | 2(+XS) 2(-)
> 19. [ ]  M19.  `content/en/docs/reference/_index.md`  | 3(+XS) 8(-)
> 20. [ ]  M20.  `content/en/docs/reference/access-authn-authz/service-accounts-admin.md`  | 47(+M) 32(-)
> 
> ### 1 files to be deleted
> 1. [ ]  D1.  `content/en/docs/concepts/extend-kubernetes/extend-cluster.md`

### CODE REVIEW

**Comment by seokho-son** [comment]
@kubernetes/sig-docs-ko-reviews 리뷰 가능하신 분~ self assign 부탁 드립니다.. ^^

File: content/ko/docs/concepts/extend-kubernetes/_index.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 title: 쿠버네티스 확장
 weight: 110
 description: 쿠버네티스 클러스터의 동작을 변경하는 다양한 방법
+feature:
+  title: 확장성을 고려하여 설계됨
+  description: >
+    쿠버네티스 업스트림 소스코드 수정 없이 쿠버네티스 클러스터에 기능을 추가할 수 있다.
```
Comment (pjhwa) [inline content/ko/docs/concepts/extend-kubernetes/_index.md:8]:
```suggestion
    쿠버네티스 업스트림 소스 코드 수정 없이 쿠버네티스 클러스터에 기능을 추가할 수 있다.
```

File: content/ko/docs/concepts/services-networking/service.md [hunk 2]
```diff
@@ -935,11 +935,18 @@ Classic ELB의 연결 드레이닝은
         # 값 보다 작아야한다. 기본값은 5이며, 2와 60 사이여야 한다.
 
         service.beta.kubernetes.io/aws-load-balancer-security-groups: "sg-53fae93f"
-        # 생성된 ELB에 추가할 기존 보안 그룹 목록.
-        # service.beta.kubernetes.io/aws-load-balancer-extra-security-groups 어노테이션과 달리, 이는 이전에 ELB에 할당된 다른 모든 보안 그룹을 대체한다.
+        # 생성된 ELB에 설정할 기존 보안 그룹(security group) 목록.
+        # service.beta.kubernetes.io/aws-load-balancer-extra-security-groups 어노테이션과 달리, 이는 이전에 ELB에 할당된 다른 모든 보안 그룹을 대체하며,
+        # '해당 ELB를 위한 고유 보안 그룹 생성'을 오버라이드한다.
+        # 목록의 첫번째 보안 그룹 ID는 인바운드 트래픽 (서비스 트래픽과 헬스 체크) 이 워커 노드로 향하도록 하는 규칙으로 사용된다.
```
Comment (pjhwa) [inline content/ko/docs/concepts/services-networking/service.md:941]:
```suggestion
        # 목록의 첫 번째 보안 그룹 ID는 인바운드 트래픽(서비스 트래픽과 헬스 체크)이 워커 노드로 향하도록 하는 규칙으로 사용된다.
```

File: content/ko/docs/concepts/services-networking/service.md [hunk 3]
```diff
@@ -935,11 +935,18 @@ Classic ELB의 연결 드레이닝은
...
+        # 여러 ELB가 하나의 보안 그룹 ID와 연결되면, 1줄의 허가 규칙만이 워커 노드 보안 그룹에 추가된다.
+        # 즉, 만약 여러 ELB 중 하나를 지우면, 1줄의 허가 규칙이 삭제되어, 같은 보안 그룹 ID와 연결된 모든 ELB에 대한 접속이 막힌다.
+        # 적절하게 사용되지 않으면 이는 다수 서비스 불가 상황을 유발할 수 있다.
```
Comment (pjhwa) [inline content/ko/docs/concepts/services-networking/service.md:944]:
```suggestion
        # 적절하게 사용되지 않으면 이는 다수의 서비스가 중단되는 상황을 유발할 수 있다.
```

- 좀 더 자연스럽게 수정했습니다.

File: content/ko/docs/concepts/services-networking/service.md [hunk 4]
```diff
@@ -935,11 +935,18 @@ Classic ELB의 연결 드레이닝은
...
 
         service.beta.kubernetes.io/aws-load-balancer-extra-security-groups: "sg-53fae93f,sg-42efd82e"
-        # ELB에 추가될 추가 보안 그룹(security group) 목록
+        # 생성된 ELB에 추가할 추가 보안 그룹 목록
+        # 이 방법을 사용하면 이전에 생성된 고유 보안 그룹이 그대로 유지되므로, 각 ELB가 고유 보안 그룹 ID와 그에 매칭되는 허가 규칙 라인을 소유하여
+        # 트래픽 (서비스 트래픽과 헬스 체크) 이 워커 노드로 향할 수 있도록 한다. 여기에 기재되는 보안 그룹은 여러 서비스 간 공유될 수 있다.
```
Comment (pjhwa) [inline content/ko/docs/concepts/services-networking/service.md:949]:
```suggestion
        # 트래픽(서비스 트래픽과 헬스 체크)이 워커 노드로 향할 수 있도록 한다. 여기에 기재되는 보안 그룹은 여러 서비스 간 공유될 수 있다.
```

File: content/ko/docs/reference/access-authn-authz/service-accounts-admin.md [hunk 5]
```diff
@@ -48,37 +49,51 @@ weight: 50
...
+            - fieldRef:
+                apiVersion: v1
+                fieldPath: metadata.namespace
+              path: namespace
+```
+
+프로젝티드 볼륨은 세 가지로 구성된다.
+
+1. kube-apiserver로부터 TokenRequest API를 통해 얻은 서비스어카운트토큰(ServiceAccountToken). 서비스어카운트토큰은 기본적으로 1시간 뒤에, 또는 파드가 삭제될 때 만료된다. 서비스어카운트토큰은 파드에 연결되며 kube-apiserver를 위해 존재한다.
+1. kube-apiserver에 대한 연결을 확인하는 데 사용되는 CA 번들을 포함하는 컨피그맵. 이 기능은 모든 네임스페이스에 "kube-root-ca.crt" 컨피그맵을 게시하는 기능 게이트인 `RootCAConfigMap`이 활성화되어 있어야 동작한다. `RootCAConfigMap`은 1.20에서 기본적으로 활성화되어 있으며, 1.21 이상에서는 항상 활성화된 상태이다.
```
Comment (pjhwa) [inline content/ko/docs/reference/access-authn-authz/service-accounts-admin.md:89]:
```suggestion
1. kube-apiserver에 대한 연결을 확인하는 데 사용되는 CA 번들을 포함하는 컨피그맵(ConfigMap). 이 기능은 모든 네임스페이스에 "kube-root-ca.crt" 컨피그맵을 게시하는 기능 게이트인 `RootCAConfigMap`이 활성화되어 있어야 동작한다. `RootCAConfigMap`은 1.20에서 기본적으로 활성화되어 있으며, 1.21 이상에서는 항상 활성화된 상태이다.
```

- 문서에서 처음 언급되어 한영병기했습니다.

**Review by pjhwa — COMMENTED** [review]
@jihoon-seo 수고 많으셨습니다! 감사합니다.
몇 가지 작은 의견들을 드립니다. 확인하시고, 선택적으로 반영하시면 될 것 같습니다. ^^

**Comment by jihoon-seo** [comment]
@pjhwa 리뷰 감사합니다!
알려 주신 사항들을 모두 반영하여 commit, squash, force-push 하였습니다.
검토를 부탁드립니다. 😊

**Comment by pjhwa** [comment]
@jihoon-seo 감사합니다!!!

/lgtm

**Comment by seokho-son** [comment]
Thank you !
/approve

**Pull request merged**